### DISCUSSION TITLE
Calculating a generic/average noise level from LW_ROADS

### DISCUSSION THREAD

**Discussion by pandanwaffles** [Universite-Gustave-Eiffel/NoiseModelling#380] [Q&A]
Hi, 

Would like to ask how I can calculate a generic/average noise level through out the day from the LW_ROADS data. Currently, these are the fields in the LW_ROADS attribute table when viewed on QGIS. I would like to get an average of the noise levels during day, evening, night and the entire day respectively to calculate noise levels from the roads at receiver points using OpeNoise plugin on QGIS. 

![image](https://user-images.githubusercontent.com/90554682/132984757-918cf391-dda1-44c2-8091-156b46f8b2eb.png)

Would appreciate some guidance in this, I'm completely new and unfamiliar to the topic on noise!

Thanks in advance!

**Comment by pierromond** [comment]
Hi !

You can calculate the LDEN value or Leq Value.

For example : 

Lden = 10Lg[(12/24).10(Ld/10)+(4/24).10((Le+5)/10)+(8/24).10((Ln+l0)/l0)]

Lden have a majoration of evening and night noise to take into account the annoyance increase over these periods

Leq = 10Lg[(12/24).10(Ld/10)+(4/24).10((Le)/10)+(8/24).10((Ln)/l0)]

I hope it anwsers to your question.

**Reply by pandanwaffles** [reply]
Thank you so much for the answer! 

Would you be able to explain what the different values indicate? For example, LWD63 vs LWD125. What does the value 63 & 125 indicate?

Please pardon me for asking!

Thank you in advance!

**Comment by pierromond** [comment]
This is the energy per frequency band. It is necessary to make the sum (energy sum) of all the frequency bands to have the global decibel.

**Reply by pandanwaffles** [reply]
Just to clarify the energy is noise energy? So for example, to get a global decibel for the LWD I'd have to sum up the value in the fields of LWD63 - LWD8000? 

Please pardon me for asking!

Thank you in advance!

**Reply by pierromond** [reply]
Yes, noise energy.

Here you can find some info : http://www.sengpielaudio.com/calculator-spl.htm

**Reply by pandanwaffles** [reply]
Alright! Thank you so much for the clarification and help!
